unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -572,7 +572,7 @@ Let $OPInclusions = [\u00A1\u00BF\u2E18\U00013258-\U0001325A\U00013286\U00013288
 
 # See L2/22-086 for an explanation of the special case of Batak.
```

Comment:
After this line please add a comment like
```
# Consensus 177-C43 changed the Batak viramas from InSC=Pure_Killer to Reordering_Killer.
```

Reply:
Done